### ISSUE TITLE
accessibilitySummary and display language

### ISSUE THREAD

**Issue by w3cbot** [w3c/i18n-activity#1542]
**This is a tracker issue.** Only discuss things here if they are i18n group internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ https://github.com/w3c/publ-a11y/issues/94

**Comment by aphillips** [comment]
Seems to have been addressed. I'm not sure that we'd be 100% satisfied, since it should be possible to label elements of a TOC with an override language (and that would affect a11y for screen readers, for example)
